frontend/src/txns/google/duration.ts

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable */
+import Long from "long";
```

Comment:
Are we using `duration` and `timestamp` file in the codebase? If not let's remove this folder.